examples/api/lib/material/menu_bar/create_material_menu.0.dart

```diff
@@ -0,0 +1,238 @@
+      case MenuSelection.hideMessage:
+        showingMessage = false;
+        break;
+      case MenuSelection.colorMenu:
+        break;
+      case MenuSelection.colorRed:
+        backgroundColor = Colors.red;
+        break;
+      case MenuSelection.colorGreen:
+        backgroundColor = Colors.green;
+        break;
+      case MenuSelection.colorBlue:
+        backgroundColor = Colors.blue;
+        break;
+    }
+  }
+
+  void _updateMenu() {
+    _menuHandle = createMaterialMenu(
+      buttonFocusNode: _buttonFocusNode,
+      controller: _controller,
+      children: <Widget>[
+        MenuItemButton(
+          child: Text(MenuSelection.about.label),
+          onPressed: () => _activate(MenuSelection.about),
+        ),
+        // Toggles the message.
+        MenuItemButton(
+          onPressed: () => _activate(MenuSelection.showMessage),
+          shortcut: MenuSelection.showMessage.shortcut,
+          child: Text(
+            showingMessage ? MenuSelection.hideMessage.label : MenuSelection.showMessage.label,
+          ),
+        ),
+        // Hides the message, but is only enabled if the message isn't already hidden.
+        MenuItemButton(
+          onPressed: showingMessage ? () => _activate(MenuSelection.resetMessage) : null,
+          shortcut: MenuSelection.resetMessage.shortcut,
+          child: Text(MenuSelection.resetMessage.label),
+        ),
+        MenuButton(
+          menuChildren: <Widget>[
+            MenuItemButton(
+              onPressed: () => _activate(MenuSelection.colorRed),
+              shortcut: MenuSelection.colorRed.shortcut,
+              child: Text(MenuSelection.colorRed.label),
+            ),
+            MenuItemButton(
+              onPressed: () => _activate(MenuSelection.colorGreen),
+              shortcut: MenuSelection.colorGreen.shortcut,
+              child: Text(MenuSelection.colorGreen.label),
+            ),
+            MenuItemButton(
+              onPressed: () => _activate(MenuSelection.colorBlue),
+              shortcut: MenuSelection.colorBlue.shortcut,
+              child: Text(MenuSelection.colorBlue.label),
+            ),
+          ],
+          child: const Text('Background Color'),
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    _updateMenu();
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        // The controlling widget for the menu is wrapped by a MenuAnchor that
+        // is given the MenuController that the menu is using. This supplies the
+        // menu with a bounding box to use when determining where it should be
+        // placed.
+        MenuAnchor(
+          controller: _controller,
+          builder: (BuildContext context) {
+            return TextButton(
+              focusNode: _buttonFocusNode,
+              onPressed: () {
+                if (_menuHandle.isOpen) {
+                  _menuHandle.close();
```

Comment:
> MenuHandles are for individual menus. MenuControllers allow you to link otherwise unrelated menus so that you can close them all at once and tab between them when they are open.

I think it would be enough to just document this in the doc comments for MenuHandle and MenuController.